README.md

```diff
@@ -85,9 +87,76 @@ model_repository/
 ```
 
 By default, the FIL backend assumes that XGBoost binary models will be named
-`xgboost.model`, XGBoost json models will be named `xgboost.json`, and LightGBM
-models will be named `model.txt`, but this can be tweaked through standard
-Triton configuration options.
+`xgboost.model`, XGBoost json models will be named `xgboost.json`, LightGBM
+models will be named `model.txt`, and Treelite binary models will be named
+`checkpoint.tl`, but this can be tweaked through standard Triton configuration
+options.
+
+#### Scikit-Learn and cuML random forest support
+
+While LightGBM and XGBoost have their own serialization formats that are
+directly supported by the Triton FIL backend, random forest models trained with
+[Scikit-Learn](https://scikit-learn.org/stable/modules/model_persistence.html)
+or [cuML](https://docs.rapids.ai/api/cuml/stable/pickling_cuml_models.html) are
+generally serialized using Python's
+[pickle](https://docs.python.org/3/library/pickle.html) module. In order to
+avoid a round-trip through Python in Triton, we instead provide scripts to
+convert pickled RF models to Treelite's binary checkpoint format. You can
+download the relevant script for Scikit-Learn [here](https://raw.githubusercontent.com/triton-inference-server/fil_backend/main/scripts/convert_sklearn)
+and for cuML
+[here](https://raw.githubusercontent.com/triton-inference-server/fil_backend/main/scripts/convert_cuml.py).
```

Comment:
Might be worth to immediately check links after merge to be sure they work well

Reply:
Will do! Thanks for the reminder